contrib/crane/template.py

```diff
@@ -0,0 +1,43 @@
+pkgname = "crane"
+pkgver = "0.20.1"
+pkgrel = 0
+build_style = "go"
+make_build_args = [
+    "./cmd/crane",
+]
+hostmakedepends = [
```

Comment:
`hostmakedepends = ["go"]`

Reply:
ok, but note `black` does not care and the current version is better for diffing changes